python/pyspark/mllib/clustering.py

```diff
@@ -106,12 +109,12 @@ def predict(self, x):
                 best_distance = distance
         return best
 
-    def computeCost(self, rdd):
+    def computeCost(self, data):
```

Comment:
I'm afraid it's too late to make changes like this to the API.  The release is already done.